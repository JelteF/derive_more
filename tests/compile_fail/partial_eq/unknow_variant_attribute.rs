#[derive(derive_more::PartialEq)]
enum Enum {
    #[partial_eq(unknown)]
    Bar { i: i32 },
}

fn main() {}
