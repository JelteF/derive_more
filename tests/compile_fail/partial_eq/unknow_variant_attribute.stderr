error: only `skip`/`ignore` allowed here
 --> tests/compile_fail/partial_eq/unknow_variant_attribute.rs:3:18
  |
3 |     #[partial_eq(unknown)]
  |                  ^^^^^^^
